### ISSUE TITLE
feat: add alternative query for Vintage Story

### ISSUE THREAD

**Issue by CosminPerRam** [gamedig/node-gamedig#608]
**What is this feature about?**
Altough we added support for this game in #606, it has been done via using the master server method, and [there is a mod](https://mods.vintagestory.at/theserverstats) for it to expose a query endpoint.

**Additional context/references**
It wouldn't hurt to provide more query methods, as master servers will eventually be shutdown.
The mod exposes a HTTP server and provides a GET endpoint which returns a json with the data, it seems pretty straight forward to add.

Original issue request for the game: #328

**Comment by dgibbs64** [comment]
Small update. Turns out Killing Floor has 3 different querys. It also has gamespy1

**Comment by tyronx** [comment]
> It wouldn't hurt to provide more query methods, as (for Vintage Story) master servers will eventually be shutdown.

Yep, this feature of yours incurred cloud hosting costs of over 5000 USD because you query this stuff so often. 96% of all queries on the master server is by gamedig. Jesus christ people. We will use every possible method available to us to block this service.

**Comment by podrivo** [comment]
OMG @tyronx! Tell us more about this! :O
